frontend/src/utils.js

```diff
@@ -21,20 +21,20 @@ export const djikstra = (gameMap) => {
 
     while (queue) {
         const { path, time } = queue.shift();
-        const [x, y] = path.at(-1);
```

Comment:
We're letting this issue propagate too far back, we should keep the coordinate conversion as close to the rendering as possible and have a nice clean separation between game coordinates and the rendering coordinates.

I haven't had a chance to put this in yet, but we should have a function like `convertGameCoordToRenderCoord`, that would take `Vector3(x,y,z)` and return `Vector3(x,z,y)` or whatever it should be, so that we never have to think of it again. If that makes sense you could add it in, otherwise I can add it in with the react three update.

Either way, I think you should leave `djikstra` as is, I don't believe there was any functional change?

Reply:
So everywhere except the renderer, x and y are horizontal and z is vertical? and all blocks are 1x1x1? Sounds good to me

Reply:
Yes, at least for the map I think integer values makes the most sense.